ext/scheduler/airflow2/tests/Makefile

```diff
@@ -1,4 +1,5 @@
 all: build
 
 test:
-	@python -m unittest discover .
```

Comment:
Lets move this target into the base one as it becomes hard for discoverability & no one runs these tests